google-ads/src/main/java/com/google/ads/googleads/lib/utils/AbstractErrorUtils.java

```diff
@@ -108,8 +110,7 @@
     List<GoogleAdsErrorT> result = new ArrayList();
```

Comment:
If we make this a Set, then we don't need the `contains` check on line 117. If you want to preserve insertion order, you could use `LinkedHashSet`.